pre-cache/olm

```diff
@@ -35,7 +35,7 @@ export_index(){
     else
         opm_bin=$image_mount/bin/opm
     fi
-    $opm_bin index export -i $index -p $packages -c podman -f $index_download_path
+    $opm_bin render $index > $rendered_index_path
```

Comment:
Just to clarify, we don't include opm binary in our container image to keep its size minimal. The host doesn't contain this binary either. However, each OLM index image we pull contains opm binary capable of serving the registry and the index content, and we are copying this binary into the pre-caching workload container. So first of all, we can't bump it.
Regarding the supportability, all four default catalogs supported by Red Hat already contain opm binary supporting the `render` subcommand already in v4.9. This subcommand is not `alpha`.  Nevertheless you've put a seed of doubt in my mind. 
Maybe it's worth to keep both variants? First try with `render`, and if it fails try with `index export`?
What do you think?
I'll try to find since when `render` is supported to help us decide.